core/src/main/java/org/apache/calcite/sql/dialect/VerticaSqlDialect.java

```diff
@@ -53,4 +57,8 @@ public VerticaSqlDialect(Context context) {
       return super.supportsFunction(operator, type, paramTypes);
     }
   }
+
+  @Override public void unparseOffsetFetch(SqlWriter writer, @Nullable SqlNode offset, @Nullable SqlNode fetch) {
```

Comment:
you need to add at least one test.

Reply:
@mihaibudiu , I'd like to, but couldn't find existing tests for thos or other `SqlDialect` classes or `unparse*()` methods, can you advise how to start?

Reply:
RelToSqlConverterTest